web/src/app/shared/services/classifications/classifications.service.ts

```diff
@@ -21,13 +22,13 @@ export class ClassificationsService {
   private classificationSaved = new Subject<number>();
   private classificationResourcePromise: Promise<ClassificationResource>;
   private lastDomain: string;
+  // TODO: this should not be here in the service
```

Comment:
We should do some research and think about this in general.
Do we want select statements within the service? If not, what are the alternatives.